package.json

```diff
@@ -1,25 +1,48 @@
 {
-  "name": "genesis-translations",
-  "version": "2.10.1",
-  "description": "Genesis Translations",
-  "bugs": {
-    "url": "https://github.com/thememix/genesis-translations/issues"
-  },
-  "homepage": "https://thememix.com/plugins/genesis-translations/",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/thememix/genesis-translations"
-  },
-  "author": "ThemeMix",
-  "license": "GPL-3.0+",
-  "devDependencies": {
-    "grunt": "^1.0.1",
-    "grunt-contrib-concat": "~1.0.0",
-    "grunt-contrib-imagemin": "~1.0.0",
-    "grunt-contrib-jshint": "~1.0.0",
-    "grunt-contrib-qunit": "~1.2.0",
-    "grunt-glotpress": "~0.2.2",
-    "grunt-jsvalidate": "~0.2.2",
-    "grunt-wp-i18n": "~0.5.4"
-  }
+	"name": "genesis-translations",
+	"version": "2.10.1",
+	"description": "Genesis Translations",
+	"bugs": {
+		"url": "https://github.com/thememix/genesis-translations/issues"
+	},
+	"homepage": "https://thememix.com/plugins/genesis-translations/",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/thememix/genesis-translations"
+	},
+	"author": "ThemeMix",
+	"license": "GPL-3.0+",
```

Comment:
SPDX identifier is invalid. Should be `GPL-3.0-or-later` (or `GPL-2.0-or-later` for it to be better compatible with Genesis and WP core).

Reply:
Thanks, @GaryJones. I changed it here and in `composer.json` to `GPL-2.0-or-later`. If @remkus wants to use `GPL-3.0-or-later` for somer reason I'm happy to make that change.

Reply:
Yeah, that's fine!